.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: java
+dist: trusty
+sudo: true
+before_cache:
+  - rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
```

Comment:
@lbergelson So can I just use the same text that is embedded in the gatk yml file ?